frontend/src/entities/configuration/dynamic-configuration.tsx

```diff
@@ -218,413 +223,520 @@ export function DynamicConfiguration () {
         setFields({ 'configuration.EMRConfig.applications': updatedSelectedApps });
     };
 
-    return (
-        <Container
-            header={
-                <Header
-                    variant='h2'
-                    description={`The current dynamic configuration of ${window.env.APPLICATION_NAME}. These settings can be modified without redeploying the application.`}
-                >
-                    {window.env.APPLICATION_NAME} Dynamic Configuration
-                </Header>
+
+    const isObject = (x) => typeof x === 'object' && !Array.isArray(x) && x !== null;
+
+    /**
+     * Computes the difference between two JSON objects, recursively.
+     *
+     * This function takes two JSON objects as input and returns a new object that
+     * contains the differences between the two. Works with nested objects.
+     *
+     * @param {object} [obj1={}] - The first JSON object to compare.
+     * @param {object} [obj2={}] - The second JSON object to compare.
+     * @returns {object} - A new object containing the differences between the two input objects.
+     */
+    function getJsonDifference (obj1 = {}, obj2 = {}) {
```

Comment:
If we are making this as a utility it should be in a utility file.

Reply:
I wasn't really planning on making this a utility. its a very specific usecase for this file. I only made it into a function so that its cleaner. I wont move it to a utility file unless you think this file is getting out of hand.